"failed to load theme" logs despite it is a standard theme

### Summary

not a problem, but for some reason these "failed to load theme" messages appear in logs on every startup for standard themes, everything works fine.

sample:

```

2484   │ 2022-08-09T17:04:12.370 helix_view::theme [WARN] Theme: malformed hexcode: grey
2485   │ 2022-08-09T17:04:12.401 helix_view::theme [WARN] Theme: malformed hexcode: grey
2486   │ 2022-08-09T17:04:12.431 helix_view::theme [WARN] Theme: malformed hexcode: grey
2487   │ 2022-08-09T17:04:12.583 helix_view::theme [WARN] Theme: malformed hexcode: purple
2488   │ 2022-08-09T17:04:12.583 helix_view::theme [WARN] Theme: malformed hexcode: light-grey
2489   │ 2022-08-09T17:04:12.583 helix_view::theme [WARN] Theme: malformed hexcode: greay
2490   │ 2022-08-09T17:04:17.849 helix_view::theme [WARN] Theme: malformed hexcode: purple
2491   │ 2022-08-09T17:04:17.849 helix_view::theme [WARN] Theme: malformed hexcode: greay
2492   │ 2022-08-09T17:04:17.849 helix_view::theme [WARN] Theme: malformed hexcode: light-grey
2493   │ 2022-08-09T17:04:28.207 helix_view::theme [WARN] Theme: malformed hexcode: greay
2494   │ 2022-08-09T17:04:28.207 helix_view::theme [WARN] Theme: malformed hexcode: purple
2495   │ 2022-08-09T17:04:28.207 helix_view::theme [WARN] Theme: malformed hexcode: light-grey
2496   │ 2022-08-09T17:04:30.236 helix_view::theme [WARN] Theme: malformed hexcode: grey
2497   │ 2022-08-09T17:04:30.763 helix_view::theme [WARN] Theme: malformed hexcode: grey
2498   │ 2022-08-09T17:04:31.502 helix_view::theme [WARN] Theme: malformed hexcode: grey
2499   │ 2022-08-09T17:04:36.329 helix_view::theme [WARN] Theme: malformed hexcode: grey
2500   │ 2022-08-09T17:04:36.359 helix_view::theme [WARN] Theme: malformed hexcode: grey
2501   │ 2022-08-09T17:04:36.390 helix_view::theme [WARN] Theme: malformed hexcode: grey
2502   │ 2022-08-09T17:04:36.542 helix_view::theme [WARN] Theme: malformed hexcode: greay
2503   │ 2022-08-09T17:04:36.542 helix_view::theme [WARN] Theme: malformed hexcode: purple
2504   │ 2022-08-09T17:04:36.542 helix_view::theme [WARN] Theme: malformed hexcode: light-grey
2505   │ 2022-08-09T17:05:07.707 helix_view::theme [WARN] Theme: malformed hexcode: grey
2506   │ 2022-08-09T17:05:09.210 helix_view::theme [WARN] Theme: malformed hexcode: grey
2507   │ 2022-08-09T17:05:10.262 helix_view::theme [WARN] Theme: malformed hexcode: grey
2508   │ 2022-08-09T17:05:12.829 helix_view::theme [WARN] Theme: malformed hexcode: purple
2509   │ 2022-08-09T17:05:12.829 helix_view::theme [WARN] Theme: malformed hexcode: greay
2510   │ 2022-08-09T17:05:12.829 helix_view::theme [WARN] Theme: malformed hexcode: light-grey
2511   │ 2022-08-09T17:05:57.096 helix_view::editor [ERROR] Failed to initialize the LSP for `source.toml` { cannot find binary path }
**2512   │ 2022-08-09T17:06:31.963 helix_term::application [WARN] failed to load theme `catppuccin-frappe` - No such file or directory (os error 2)**
```

### Reproduction Steps

<!-- Ideally provide a key sequence and/or asciinema.org recording. --> 

I tried this:

1. `hx`

I expected this to happen:

Instead, this happened:


### Helix log

_No response_

### Platform

Linux

### Terminal Emulator

alacritty 0.10.1

### Helix Version

helix 22.05 (bf6f7042)

--- a/runtime/themes/ayu_dark.toml
+++ b/runtime/themes/ayu_dark.toml
@@ -1,5 +1,5 @@
 # Author: André Sá <enkodr@outlook.com>
-# Based on the AYU theme colors from https://github.com/dempfi/ayu 
+# Based on the AYU theme colors from https://github.com/dempfi/ayu
 
 # Syntax highlighting
 "type" = { fg = "blue" }
@@ -23,7 +23,7 @@
 "namespace" = { fg = "blue" }
 "markup.heading" = { fg = "orange" }
 "markup.list" = { fg = "yellow" }
-"markup.raw.block" = { bg = "grey", fg = "orange" } 
+"markup.raw.block" = { bg = "gray", fg = "orange" }
 "markup.link.url" = { fg = "blue" }
 "markup.link.text" = { fg = "yellow" }
 "markup.link.label" = { fg = "green" }

--- a/runtime/themes/ayu_light.toml
+++ b/runtime/themes/ayu_light.toml
@@ -1,5 +1,5 @@
 # Author: André Sá <enkodr@outlook.com>
-# Based on the AYU theme colors from https://github.com/dempfi/ayu 
+# Based on the AYU theme colors from https://github.com/dempfi/ayu
 
 # Syntax highlighting
 "type" = { fg = "blue" }
@@ -23,7 +23,7 @@
 "namespace" = { fg = "blue" }
 "markup.heading" = { fg = "orange" }
 "markup.list" = { fg = "yellow" }
-"markup.raw.block" = { bg = "grey", fg = "orange" } 
+"markup.raw.block" = { bg = "gray", fg = "orange" }
 "markup.link.url" = { fg = "blue" }
 "markup.link.text" = { fg = "yellow" }
 "markup.link.label" = { fg = "green" }

--- a/runtime/themes/ayu_mirage.toml
+++ b/runtime/themes/ayu_mirage.toml
@@ -1,5 +1,5 @@
 # Author: André Sá <enkodr@outlook.com>
-# Based on the AYU theme colors from https://github.com/dempfi/ayu 
+# Based on the AYU theme colors from https://github.com/dempfi/ayu
 
 # Syntax highlighting
 "type" = { fg = "blue" }
@@ -23,7 +23,7 @@
 "namespace" = { fg = "blue" }
 "markup.heading" = { fg = "orange" }
 "markup.list" = { fg = "yellow" }
-"markup.raw.block" = { bg = "grey", fg = "orange" } 
+"markup.raw.block" = { bg = "gray", fg = "orange" }
 "markup.link.url" = { fg = "blue" }
 "markup.link.text" = { fg = "yellow" }
 "markup.link.label" = { fg = "green" }

--- a/runtime/themes/base16_transparent.toml
+++ b/runtime/themes/base16_transparent.toml
@@ -2,10 +2,10 @@
 
 "ui.background" = { fg = "white"}
 "ui.background.separator" = { fg = "gray" }
-"ui.menu" = { fg = "greay" }
+"ui.menu" = { fg = "gray" }
 "ui.menu.selected" = { modifiers = ["reversed"] }
 "ui.menu.scroll" = { fg = "light-gray" }
-"ui.linenr" = { fg = "light-grey" }
+"ui.linenr" = { fg = "light-gray" }
 "ui.linenr.selected" = { fg = "white",  modifiers = ["bold"] }
 "ui.popup" = { fg = "white" }
 "ui.window" = { fg = "white" }
@@ -50,7 +50,7 @@
 "markup.raw" = "green"
 "markup.normal" = { fg = "blue" }
 "markup.insert" = { fg = "green" }
-"markup.select" = { fg = "purple" }
+"markup.select" = { fg = "magenta" }
 
 "diff.plus" = "light-green"
 "diff.delta" = "yellow"

